app/controllers/application_controller.rb

```diff
@@ -22,12 +23,11 @@ def append_to_log(params)
   # Rails' instrumentation code, and is run after each request.
   def append_info_to_payload(payload)
     super
-    if @custom_log_items
-      payload[:custom_log_items] = @custom_log_items
-    end
-  end
 
-  # :nocov:
+    append_to_log(request_id: RequestStore.store[:request_id])
```

Comment:
+1 for a comment r.e. private method – the mechanism for extending the rails request log is surprisingly poor. However given that this is an overridden method surely it makes sense for its implementation to be as minimal as possible?

This method really has nothing to do with the request_id – it only has to do with generic logging. There are other places that call `append_to_log` outside this method, as is intended. Using an API as designed is not indirection. The whole point of `append_to_log` is that you _don't_ have to put your logging inside this private method.